pkgs/applications/audio/cozy/default.nix

```diff
@@ -39,6 +40,15 @@ python3Packages.buildPythonApplication rec {
     gobject-introspection
   ];
 
+  patches = [
+    (fetchpatch  {
+        name = "invalid-version-gsettings";
+        url = "https://patch-diff.githubusercontent.com/raw/geigi/cozy/pull/762.patch";
```

Comment:
can you fetch the commit instead of the pull request? I know for sure that this kind of urls change when the pull request is force-pushed, and maybe it disappears when users are deleted.